pkgs/tools/z/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, lib, ... }:
```

Comment:
`lib` can be dropped here when the change to meta is done. I don't think ellipsis (`...`) is commonly used in package derivations so it should probably be removed.